### DISCUSSION TITLE
Where do I read the sponsors' monthly newsletters?

### DISCUSSION THREAD

**Discussion by moodmosaic** [community/community#42136] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

When I sponsor an account on GitHub, where am I supposed to read its monthly newsletters? I don't receive any notifications on the GitHub website or on my email account (linked with my GitHub account).

**Comment by getlost01** [comment, marked as answer]
When you sponsor an account on GitHub, the account owner will typically send newsletters to their sponsors through email. These newsletters should be sent to the email address associated with your GitHub account.

If you are not receiving the newsletters, there are a few things you can try:

1. Check your spam or junk mail folder. Sometimes, newsletters can get caught in the spam or junk mail filter.
2. Make sure that your email address is correctly linked to your GitHub account. You can check this by going to your account settings on GitHub and verifying that the correct email address is listed.
3. Check that you have not unsubscribed from the newsletters. Some accounts may give you the option to unsubscribe from their newsletters. If you have unsubscribed, you will not receive the newsletters.
4. Contact the account owner. If you are still having trouble receiving the newsletters, you can try contacting the account owner directly to ask if they are having any issues with their email list or if there is a problem with your subscription.

**Reply by moodmosaic** [reply]
Thank you 👍 It was `3.` in my case.

**Reply by donmccurdy** [reply]
It's highly worth noting that "Receive email updates from [account]" is *off* by default, you have to explicitly opt in to receiving communication from the account you are sponsoring.
